src/Microsoft.DotNet.ProjectJsonMigration/Rules/TemporaryMutateProjectJsonRule.cs

```diff
@@ -39,8 +40,8 @@ public void Apply(MigrationSettings migrationSettings, MigrationRuleInputs migra
             }
 
             var json = CreateDestinationProjectFile(sourceProjectFile, destinationProjectFile);
-            InjectSdkReference(json, ConstantPackageNames.CSdkPackageName, migrationSettings.SdkPackageVersion);
             RemoveRuntimesNode(json);
+            RemoveAllDependencies(json);
```

Comment:
Why?  I would think now we wouldn't need to mutate the project.json at all anymore.